en/components/toggle.md

```diff
@@ -69,23 +82,27 @@ export class Class {
 
 If all went well, you should see the following in your browser:
 
-<div class="sample-container" style="height: 200px">
-Demo
+<div class="sample-container loading" style="height: 370px">
+    <iframe id="toggle-sample-1-iframe" src='{environment:demosBaseUrl}/toggle-sample-1' width="100%" height="100%" seamless frameBorder="0" onload="onSampleIframeContentLoaded(this);"></iframe>
+</div>
+<div>
+    <button data-localize="stackblitz" class="stackblitz-btn" data-iframe-id="toggle-sample-1-iframe" data-demos-base-url="{environment:demosBaseUrl}">                       view on stackblitz
+    </button>
 </div>
 
-#### Automatic toggle actions
+### Automatic toggle actions
 
-In order to prevent this invocation of these methods there is a directive which has `onClick` handler and changes the state to the toggle we are referred to. So let's dive in. Firstly we have to import the **IgxToggleActionModule** in our **app.module.ts**
+In order to prevent this invocation of these methods there is a directive which has `onClick` handler and changes the state to the toggle we are referred to. So let's dive in. To take advantage of this functionality we will have to use **IgxToggleActionDirective** Which is located into the same **IgxToggleModule**.
```

Comment:
"we will have to use **IgxToggleActionDirective**".  I don't understand what "Which is located into the same **IgxToggleModule** means.  Do you mean it's imported from the same source as IgxToggleModule?

Reply:
Yes, that's what I meant.  I have corrected this sentence within the latest commit.